flexmeasures/data/queries/annotations.py

```diff
@@ -8,26 +8,33 @@
 from flexmeasures.data.models.annotations import (
     Annotation,
     GenericAssetAnnotationRelationship,
+    SensorAnnotationRelationship,
 )
 from flexmeasures.data.models.data_sources import DataSource
 
 
 def query_asset_annotations(
-    asset_id: int,
+    asset_or_sensor_id: int,
+    relationship_module,
```

Comment:
Missing type annotation, possibly an Enum with the 3 relationship choices we currently have (with Accounts, GenericAssets and Sensors).